pkg/oauth/api/validation/validation_test.go

```diff
@@ -305,6 +349,39 @@ func TestValidateAuthorizeTokens(t *testing.T) {
 			T: field.ErrorTypeForbidden,
 			F: "metadata.namespace",
 		},
+		"no scope handler": {
+			Token: oapi.OAuthAuthorizeToken{
+				ObjectMeta: api.ObjectMeta{Name: "authorizeTokenNameWithMinimumLength"},
+				ClientName: "myclient",
+				UserName:   "myusername",
+				UserUID:    "myuseruid",
+				Scopes:     []string{"invalid"},
+			},
+			T: field.ErrorTypeInvalid,
+			F: "scopes[0]",
+		},
+		"bad scope": {
+			Token: oapi.OAuthAuthorizeToken{
+				ObjectMeta: api.ObjectMeta{Name: "authorizeTokenNameWithMinimumLength"},
+				ClientName: "myclient",
+				UserName:   "myusername",
+				UserUID:    "myuseruid",
+				Scopes:     []string{"user:dne"},
+			},
+			T: field.ErrorTypeInvalid,
+			F: "scopes[0]",
+		},
+		"illegal character": {
+			Token: oapi.OAuthAuthorizeToken{
+				ObjectMeta: api.ObjectMeta{Name: "authorizeTokenNameWithMinimumLength"},
+				ClientName: "myclient",
+				UserName:   "myusername",
+				UserUID:    "myuseruid",
+				Scopes:     []string{`user:asdf"`},
+			},
+			T: field.ErrorTypeInvalid,
+			F: "scopes[0]",
+		},
```

Comment:
Not a blocker to the review, but I notice that the tests here only check for error-cases with the scopes. It might be nice to see a success case included as well for more complete coverage.

Otherwise, LGTM.